system_tests/bigtable.py

```diff
@@ -371,31 +370,8 @@ def test_read_large_cell_limit(self):
         self.assertEqual(partial_row_data.row_key, ROW_KEY)
         cell = partial_row_data.cells[COLUMN_FAMILY_ID1]
         column = cell[COL_NAME1]
-        value = column[0].value
-        self.assertIsNotNone(value)
-        self.assertEqual(len(value), number_of_bytes)
-
-    def test_read_large_cells_over_limit(self):
-        row = self._table.row(ROW_KEY)
-        self.rows_to_delete.append(row)
-
-        data = b'1' * 51 * 1024 * 1024  # 51MB.
-        row.set_cell(COLUMN_FAMILY_ID1, COL_NAME1, data)
-        row.commit()
-        row.set_cell(COLUMN_FAMILY_ID1, COL_NAME1, data)
-        row.commit()
-
-        with self.assertRaises(_Rendezvous):
-            self._table.read_row(ROW_KEY)
-
-    def test_read_large_cell_over_limit(self):
-        row = self._table.row(ROW_KEY)
-        self.rows_to_delete.append(row)
-
-        data = b'1' * 101 * 1024 * 1024  # 101MB of 1's.
-        row.set_cell(COLUMN_FAMILY_ID1, COL_NAME1, data)
-        with self.assertRaises(_Rendezvous):
-            row.commit()
+        self.assertEqual(len(column), 1)
+        self.assertEqual(len(column[0].value), number_of_bytes)
```

Comment:
I wasn't sure if that would be slower with larger values but I guess it wouldn't be any different calling `len()` on it.